src/common-rest-api/src/main/java/org/niis/xroad/restapi/config/NotificationApiDisableSessionTimeoutFilter.java

```diff
@@ -55,7 +55,7 @@
 @Slf4j
 @Configuration
 @Order(Ordered.HIGHEST_PRECEDENCE)
-public class SessionTimeoutFilter extends GenericFilterBean {
+public class NotificationApiDisableSessionTimeoutFilter extends GenericFilterBean {
```

Comment:
Is there another `SessionTimeoutFilter` or why the complex class name?

Reply:
Yes, src/common-ui/src/main/java/ee/ria/xroad/commonui/SessionTimeoutFilter.java